demo_data/.well-known/product-passport/products/fzsonick-sodium-nickel-chloride.json

```diff
@@ -0,0 +1,20 @@
+{
+  "names": {
+    "en_US": "Sandvik Salt Battery"
```

Comment:
Has this name been confirmed? Looks wrong to me, I think it's the wrong company (also used in the filename).

Reply:
@TheHelias are you going to answer?

Reply:
This filename is clearly not for a sandvik salt battery, this will not be approved in this form.

Reply:
I mentioned on Slack that I used the names in the design pending the time the actual demo data is stated or agreed on.